<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrgenUi</class>
 <widget class="QMainWindow" name="UrgenUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_extra">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>路网</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_good">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>无漏</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_press">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>抗压</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_correct">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修正分(额外出怪)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scoreWord">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>本关得分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>191</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>UrgenUi</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_extra</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UrgenUi</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_good</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UrgenUi</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_press</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UrgenUi</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UrgenUi</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
